string status
float64 move_base_state